Change AC_Fence to get home distance from AHRS

peterbarker opened the pull request:
Closes #7408

OXINARF commented on libraries/AC_Fence/AC_Fence.cpp:
```diff
@@ -241,6 +241,12 @@ bool AC_Fence::check_fence_circle()
 
+    Vector2f home;
+    if (_ahrs.get_relative_position_NE_home(home)) {
+        // we (may) remained breached if we can't update home
```
remained -> remain?

*We may remained breached* sounds strange to me. But *We may remain breached* and *We remain breached* looks good,

peterbarker commented:
On Thu, 14 Dec 2017, Francisco Ferreira wrote:

> remained -> remain?
> 
> We may remained breached sounds strange to me. But We may remain breached
> and We remain breached looks good,

Whoops.  I was playing with that to get the correct meaning and goofed.

Fixed, rebased and pushed up.

OXINARF commented:
Merged, thanks!
